scripts/render-metrics-gif.py

```
@@ -385,25 +421,77 @@ def draw_frame(records, up_to, title, limits, kpi_sidecar=None, width=1280, heig
+
+        spark_entries = _sparkline(entries_series)
+        spark_share = _sparkline(share_series)
+
+        current_total = next(
+            (v for v in reversed(entries_series) if v is not None), 0
+        )
+        current_share = next(
+            (v for v in reversed(share_series) if v is not None), 0.0
+        )
+        line1 = f"NN  {spark_entries} {current_total:>5}"
+        line2 = f"1918{spark_share} {int(round(current_share * 100)):>4}%"
```

Ruff (0.15.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)